utils/calculate-price.js

```diff
@@ -0,0 +1,17 @@
+const Currency = require('../modules/currency/currency.model');
+
+const calculatePrice = async (price) => {
```

Comment:
We already have the same currency util in currency module. Please delete it from modules and reuse yours

Reply:
Fixed